compiler/luci/pass/src/QuantizeWeights.h

```diff
@@ -29,14 +29,16 @@ namespace luci
  */
 struct QuantizeWeights final : public luci::CircleNodeMutableVisitor<void>
 {
-  QuantizeWeights(loco::DataType input, loco::DataType output, QuantizationGranularity gr)
-    : input_type(input), output_type(output), granularity(gr)
+  QuantizeWeights(loco::DataType input, loco::DataType output, QuantizationGranularity gr,
+                  bool is_save_min_max)
+    : input_type(input), output_type(output), granularity(gr), save_min_max(is_save_min_max)
```

Comment:
The change seems unnecessary. PTAL https://github.com/Samsung/ONE/pull/11730/files#r1362939612